src/lib/vast_elements/media_files.js

```diff
@@ -27,7 +27,7 @@ class MediaFile {
   }
 
   isVideoType() {
-    return this.mimeType().match(/^video\//);
+    return this.mimeType().match(/^video|application\//);
```

Comment:
To be pedantic about matching with the other one, I think it might need `/^video\/|application\//`, (with the extra `/` at the end of `video`, since you want to match things starting with `video/`, based on the old line, and not just `video`).

Reply:
Good catch, thank you!